src/common/components/layouts/pages/upload/dropzone/DropzoneText.js

```diff
@@ -8,16 +10,20 @@ function DropzoneText(props) {
     const { files } = props;
 
     return (
-        <section className="dropzone-text">
+        <>
```

Comment:
There should be only one section that wrap all text in this component. File name and file size should be wrapped in <span> instead.